src/components/ScanTotal/ScanTotal.js

```diff
@@ -30,6 +33,7 @@ class ScanTotal extends React.Component {
           </div>}
         <div>
           <Button
+            data-testid="endSessionButton"
```

Comment:
According to testing library recommendations, getting an element by test id would be the last way to retrieve a dom element.
[https://testing-library.com/docs/queries/bytestid/ https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change for guidelines and recommendations]
Could you please 
1. try to retrieve the button by a combination of its role and name? 
2. remove the attribute data-testid?
3. adjust the test accordingly?

Reply:
@Terala-Priyanka I am not sure that it is a good idea to change it.
We can have a few buttons with the same label/role. But if we use `data-testid` it is a unique value.
Furthermore, we always try to use `data-testid` in our modules if it is possible.

cc @Dmitriy-Litvinenko

Reply:
@artem-blazhko 
The queries in testing library help in finding the elements like how any real time user would find the same element.
I think it will be good to use data-testid as only last option. I think, here you can use getByRole and name.

Reply:
_"Furthermore, we always try to use data-testid in our modules if it is possible."_ Do you mean that this is a general practice within folio modules?

Reply:
As discussed,
It is a recommended best practice.
But I am fine if you want to merge the changes.